package.json

```diff
@@ -167,6 +167,7 @@
     "csstype": "^3.0.8",
     "eslint": "5.11.1",
     "eslint-config-sentry-app": "1.74.0",
+    "eslint-plugin-escompat": "^3.1.0",
```

Comment:
@narsaynorath looks like this isnt the right plugin. From the [docs](https://www.npmjs.com/package/eslint-plugin-escompat):
> This is useful if you intend to ship code **without first using a transpiler, such as [Babel](https://babeljs.io/).

iirc we do use babel, so eslint-plugin-compat would be the right choice here

Reply:
Ah, that's what I get for throwing up a PR too early lol

I think this is the correct plugin? https://www.npmjs.com/package/eslint-plugin-compat

I will probably have to revisit this on Monday, but I appreciate you catching that and I can also find out who would have answers to whether we should add it to that repo! Now I'm realizing you had that in mind because you mentioned bumping a version and I think it makes sense as well